internal/runbits/runtime/runtime.go

```diff
@@ -53,5 +53,8 @@ func NewFromProject(
 			return nil, locale.WrapError(err, "err_update_runtime", "Could not update runtime installation.")
 		}
 	}
+	if rti.Disabled() {
+		out.Notice(locale.T("notice_runtime_disabled"))
+	}
```

Comment:
Let's put this in the same place as the behavior: https://github.com/ActiveState/cli/blob/cfbe41918961fc22599d6f8fb2bc1f1efe55314d/pkg/platform/runtime/runtime.go#L101

Reply:
But we never get to this code path because a disabled runtime never returns the NeedsUpdateError error that would trigger a call to Update().